fairlearn/reductions/_grid_search/grid_search.py

```diff
@@ -90,12 +90,12 @@ def __init__(self,
         self.grid_offset = grid_offset
         self.grid = grid
 
-        self._best_grid_index = None
-        self._predictors = []
-        self._lambda_vecs = pd.DataFrame()
-        self._objectives = []
-        self._gammas = pd.DataFrame()
-        self._oracle_execution_times = []
+        self.best_grid_index_ = None
```

Comment:
since we're in business of renaming things. i think i might prefer `best_index_` or even `best_idx_` over `best_grid_index` (is it an index of the best grid?)

Reply:
renamed to `best_idx_`